### ISSUE TITLE
Show isochrones for different departure times rather than different cutoffs

### ISSUE THREAD

**Issue by abyrd** [conveyal/analyst-server#214]
Vincent Tinet of AGAM has the following suggestion: It would be interesting to animate the isochrones for different departure times throughout the time window, for a given travel time cutoff value.

**Comment by mattwigway** [comment]
Agreed, that would be really cool. You'd really be able to see the impacts of frequency. We'd have to send back the entire propagatedtimesstore for this to work, which isn't necessarily outside the realm of possiblity, esp. if we switch to PNGs for moving isochrone data. 120 grayscale PNGs would be pretty small; bandwidth in the datacenter isn't really a problem, it's just a matter of whether we can ship them to the browser fast enough.
